docs/source/pillars/data_management.md

```diff
@@ -188,10 +188,10 @@ The ability to query and browse the data within an environment at various levels
   - Guidance
   - Importance
 * - 3.4.1.
-  - You could make a catalogue of sensitive data that you make available to users.
-  - This is particularly relevant for TREs that are an interface to a common data collection.
-    This may not be appropriate for TREs where each project has its own data sharing agreement with one or more {ref}`information asset owners <data_roles>`.
-  - Optional
+  - You should provide an open metadata catalogue of available datasets for users.
+  - This is particularly relevant for TREs with population-level data collection of general interest.
+    This may not be appropriate for TREs where each project has its own data sharing agreement with one or more {ref}`information asset owner <data_roles>` or very sensitive datasets.
```

Comment:
The `information asset owner` can delegate decisions on who can access it to another actor, so something like `data provider` makes sense. On the other hand this is guidance and is an example rather than a definitive statement, so maybe it doesn't matter?